tutoraspects/patches/k8s-deployments

```diff
@@ -492,27 +495,46 @@ spec:
             - name: SYSFS_ROOT
               value: /host/sys
             - name: VECTOR_LOG
-              value: warn
+              value: info
           volumeMounts:
             - name: data
               mountPath: /vector-data-dir
             - name: var-log
               mountPath: /var/log/
-              readOnly: true
             - mountPath: /etc/vector/vector.toml
               name: config
               subPath: k8s.toml
               readOnly: true
+            - mountPath: /var/lib
```

Comment:
Can you comment why we need these if we need to keep them?

Reply:
Those come by default from the vector-agent helm chart. I will remove it